sechub-commons-model/src/main/java/com/mercedesbenz/sechub/commons/model/SecHubConfigurationModelValidationError.java

```diff
@@ -66,7 +66,15 @@ public enum SecHubConfigurationModelValidationError {
 
     METADATA_LABEL_KEY_CONTAINS_ILLEGAL_CHARACTERS("Meta data label key contains illegal characters."),
 
-    ;
+    REMOTE_DATA_MULTI_CONFIGURATION_NOT_ALLOWED("Remote data configuration is not allowed for multiple data configurations."),
```

Comment:
Sorry but the text is a little bit hard to understand (at least two parts are tested: remote only once (per unique id?) and no filesystem with remote combined possible